commit 3cdb99d8bb9d0008b2b297080e61d6c10dd66cc8
Author: Frédéric Bérat <fberat@redhat.com>
Date:   Tue Mar 11 10:40:11 2025 +0100

    Add missing guards in include/arpa/inet.h
    
    Add the missing guards in the header, similarly to other headers at the
    same level
    
    Reviewed-by: Florian Weimer <fweimer@redhat.com>

diff --git a/include/arpa/inet.h b/include/arpa/inet.h
index d1ea13bb19d4a497..d9e55a3c7f2db9f2 100644
--- a/include/arpa/inet.h
+++ b/include/arpa/inet.h
@@ -1,3 +1,5 @@
+#ifndef _ARPA_INET_H
+/* Note: _ARPA_INET_H is defined by inet/arpa/inet.h below.  */
 #include <inet/arpa/inet.h>
 
 #ifndef _ISOMAC
@@ -17,3 +19,4 @@ libc_hidden_proto (inet_netof)
 extern __typeof (inet_network) __inet_network;
 libc_hidden_proto (__inet_network)
 #endif
+#endif
